pkg/comp-functions/functions/common/non_SLA_prom_rules.go

```diff
@@ -0,0 +1,162 @@
+package common
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"strings"
+
+	fnproto "github.com/crossplane/function-sdk-go/proto/v1beta1"
+	promV1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"github.com/vshn/appcat/v4/pkg/comp-functions/runtime"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	controllerruntime "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	SYN_TEAM          string = "schedar"
```

Comment:
Why are you suddenly using another naming scheme for variables? For golang variables are always CamelCase.

Also, you don't need to specify that it's a string.

Reply:
fixed